### PR TITLE
dotCMS/core#15955 SPAEM: Style header and title

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/Components/Header.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+header.header__menu {
```
Comment (fmontes) [inline src/Components/Header.css]: no need `header.xxx`

File: src/Components/Header.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    bottom: 0;
+    height: 500px;
+    left: 0;
+    opacity: 0.95;
+    position: absolute;
+    right: 0;
+    top: 0;
+}
+
+.navbar-light a.navbar-brand {
```
Comment (fmontes) [inline src/Components/Header.css]: Should we create a `navbar.css`?

File: src/Components/Header.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+@media screen and (max-width: 768px) {
+    .header__page-title {
+        font-size: 2.125rem;
+    }
+}
+
+.header__breadcrumbs {
+    color: var(--white);
+    font-family: 'Nunito', Arial, sans-serif;
```
Comment (fmontes) [inline src/Components/Header.css]: Isn't this the default from `body`?

File: src/Components/Nav/Nav.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+div.nav-menu__links {
```
Comment (fmontes) [inline src/Components/Nav/Nav.css]: no need tag

File: src/Components/Nav/Nav.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    flex-grow: unset;
+}
+
+button.nav-menu__toggle-button.navbar-toggler {
```
Comment (fmontes) [inline src/Components/Nav/Nav.css]: no need tag

File: src/Components/Nav/NavDropDown.js [hunk 6]
```diff
@@ -6,7 +6,7 @@ import { Link } from 'react-router-dom';
 const NavDropDown = ({ options }) => {
     return (
         <UncontrolledDropdown nav inNavbar>
-            <DropdownToggle nav caret>
+            <DropdownToggle nav caret className="nav-option">
```
Comment (fmontes) [inline src/Components/Nav/NavDropDown.js]: We use this class here but is defined in another css component, should we group all nav css into one file?

File: src/Components/News/NewsList.css [hunk 7]
```diff
@@ -4,4 +4,14 @@
...
+}
+
+.news-list__image:after {  
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100px;
+    height: 30px;
+    background-color: #fff;
```
Comment (fmontes) [inline src/Components/News/NewsList.css]: var?

File: src/Components/News/NewsList.css [hunk 8]
```diff
@@ -4,4 +4,14 @@
     grid-row-gap: 2em;
     grid-template-columns: repeat(auto-fit, minmax(18.75rem, 1fr));
     margin-top: 7em;
-}
\ No newline at end of file
+}
+
+.news-list__image:after {
```
Comment (fmontes) [inline src/Components/News/NewsList.css]: what is adding this?

File: src/Components/Header.css (hunk 2 above)
Comment (fmontes) [reply src/Components/Header.css]: Isn't this bootstrap class names?

File: src/Components/Header.css (hunk 3 above)
Comment (alfredo-dotcms) [reply src/Components/Header.css]: no, default is
`font-family: 'Poppins', Arial, sans-serif;`

File: src/Components/News/NewsList.css (hunk 8 above)
Comment (alfredo-dotcms) [reply src/Components/News/NewsList.css]: image

hides broken img icon

**Pull request merged**